moment_kinetics/test/gyroaverage_tests.jl

```diff
@@ -107,14 +107,9 @@ function gyroaverage_test(absolute_error; rhostar=0.1, pitch=0.5, ngrid=5, kr=2,
         gyrophase_nelement_global = gyrophase_nelement_local # total number of elements 
         gyrophase_discretization = "finite_difference"
         gyrophase_L = 2.0*pi
+        gyrophase_bc = "periodic"

```

Comment:
Was adding this boundary condition string necessary? I don't think any calculus functions are used for this coordinate.

Reply:
Periodic is what was being set before (as a hard-coded argument to `grid_input`), so I didn't change it. It seemed more in keeping with the style of code in this function to include it like this.